sdk/confidentialledger/azure-confidentialledger/samples/verify_service_async.py

```diff
@@ -0,0 +1,115 @@
+# -------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for
+# license information.
+# --------------------------------------------------------------------------
+
+"""
+FILE: verify_service_async.py
+DESCRIPTION:
+    This sample demonstrates methods that may be used to check conditions about the Confidential
+    Ledger service. This sample will print the results of verification methods.
+USAGE:
+    python verify_service_async.py
+    Set the environment variables with your own values before running the sample:
+    1) CONFIDENTIALLEDGER_ENDPOINT - the endpoint of the Confidential Ledger.
+"""
+
+import asyncio
+import logging
+import os
+import sys
+import tempfile
+
+from azure.confidentialledger.aio import ConfidentialLedgerClient
+from azure.confidentialledger.certificate.aio import (
+    ConfidentialLedgerCertificateClient,
+)
+from azure.identity.aio import DefaultAzureCredential
+
+
+logging.basicConfig(level=logging.ERROR)
+LOG = logging.getLogger()
+
+
+async def main():
+    # Set the values of the client ID, tenant ID, and client secret of the AAD application as
+    # environment variables:
+    #   AZURE_CLIENT_ID, AZURE_TENANT_ID, AZURE_CLIENT_SECRET, CONFIDENTIALLEDGER_ENDPOINT
+    try:
+        ledger_endpoint = os.environ["CONFIDENTIALLEDGER_ENDPOINT"]
```

Comment:
I think you can just rely on the `KeyError` here, i.e. move it out of the try except

Reply:
I followed one of the existing samples. Additionally, having run these samples myself several times while forgetting to set the environment variable, I do prefer the explicit messaging telling me what environment variable is supposed to be set. A `KeyError` would be more opaque.